app/components/login_button_component.rb

```diff
@@ -0,0 +1,26 @@
+class LoginButtonComponent < ButtonComponent
+  attr_reader :action, :color, :logo_path, :tag_options
+
+  def initialize(color: "primary", **tag_options)
+    super(
+      color: color,
+      **tag_options
+    )
+
+    @color = color
+  end
+
+  def logo_path
+    return "logo-white.svg" if color == "primary darker"
+    "logo.svg"
+  end
+
+  def css_class
+    classes = super || ['usa-button', *tag_options[:class]]
+    classes << 'login-button login-button--primary' if color == "primary"
+    classes << 'login-button login-button--primary-lighter' if color == "primary lighter"
+    classes << 'login-button login-button--primary-darker' if color == "primary darker"
```

Comment:
I think @zachmargolis may have made a similar comment earlier, but it'd be nice to just pass the raw value into the class name rather than having a different, space-separated token.

```suggestion
    classes << "login-button login-button--#{color}"
```

We'd probably want to do some validation on expected color types, similar to what we have in `AlertComponent`

https://github.com/18F/identity-idp/blob/b733821dea53bbec5aeb006c393d29a153ce3dbd/app/components/alert_component.rb#L7-L9